lib/package.json

```diff
@@ -13,15 +13,15 @@
   "author": "Backpack Design System <backpack@skyscanner.net>",
   "license": "Apache-2.0",
   "dependencies": {
-    "bpk-svgs": "^12.1.14",
-    "bpk-tokens": "^36.0.0",
     "lodash": "^4.17.15",
     "recyclerlistview": "^2.0.12",
     "theming": "^1.3.0",
     "tinymask": "^1.0.2"
   },
   "peerDependencies": {
     "@skyscanner/react-native-transitiongroup": "^1.1.3",
+    "bpk-svgs": "^12.1.14",
```

Comment:
Don't these need to be installed as devDeps too?

Reply:
They are in the root `package.json`, which are also included in this PR 😄

Reply:
Ignore me - they are!